manifests/06_deployment.yaml

```diff
@@ -49,6 +51,10 @@ spec:
         configMap:
           defaultMode: 440
           name: openshift-authentication-operator-config
+      - name: serving-cert
+        secret:
+          secretName: openshift-authentication-operator-serving-cert
+          optional: true
```

Comment:
Does this get reloaded if we start before we have it?  We probably want to wait for it otherwise.

Reply:
I think the server dynamically reloads it (for library-go metrics servers)